src/web/src/App.vue

```diff
@@ -8,10 +8,16 @@
 export default {
   name: "App",
   components: {},
+  created() {
+    if (this.$cookies.get("darkMode") == "true") {
+      this.$store.commit("TOGGLE_DARK_MODE");
```

Comment:
now that you've declared the mutation type as a constant, you should import it and use it in lieu of the string